ion_phys/examples/M1_transitions.py

```diff
@@ -0,0 +1,22 @@
+from ion_phys.ions.ca43 import Ca43, ground_level
+import scipy.constants as sc
+
+uB = sc.physical_constants['Bohr magneton'][0]
+
+
+if __name__ == '__main__':
+    # all seems correct (e.g. agrees with TPH thesis table E.4)
+    ion = Ca43(level_filter=[ground_level])
+    ion.setB(146.0942e-4)
```

Comment:
I wonder if we should consider making the clock qubit fields constants that can be imported from Ca43?